setup/configs/docker/README.md

```diff
@@ -0,0 +1,6 @@
+# docker
+
+Configuration files for docker
+
+`daemon.json` is docker daemon setup for basic gvisor runtime usage
+`daemon_passthrough.json` is the same as `daemon.json` but configured to allow network passthrough as the risk of reduced isolation
```

Comment:
```suggestion
`daemon_passthrough.json` is the same as `daemon.json` but configured to allow network passthrough at the risk of reduced isolation
```

perhaps just specify that this is done through using host network mode, "network passthrough" seemed a little confusing (ie rename this `daemon_hostnetwork.json`)